docker_daemon/README.md

```diff
@@ -0,0 +1,211 @@
+
+Some options have moved from `docker_daemon.yaml` to the main `datadog.yaml`:
+
+  * `collect_labels_as_tags` has been renamed `docker_labels_as_tags` and now supports high cardinality tags, see the details in `datadog.yaml.example`.
+  * `exclude` and `include` lists have been renamed `ac_include` and `ac_exclude`. To make filtering consistent across all components of the Agent, filtering on arbitrary tags has been dropped. The only supported filtering tags are `image` (image name) and `name` (container name). Regexp filtering is still available, see `datadog.yaml.example` for examples.
+  * The `docker_root` option has been split in two options `container_cgroup_root` and `container_proc_root`.
+  * `exclude_pause_container` has been added to exclude paused containers on Kubernetes and Openshift (defaults to true). This avoids removing them from the exclude list by error.
+
+Additional changes:
+
+  * The `TAGS` environment variable was renamed to `DD_TAGS`.
+  * The Docker Hub repository has changed from [datadog/docker-dd-agent][16] to [datadog/agent][17].
+
+The [`import`][18] command converts the old `docker_daemon.yaml` to the new `docker.yaml`. The command also moves needed settings from `docker_daemon.yaml` to `datadog.yaml`.
+
+## Data Collected
+### Metrics
+See [metadata.csv][19] for a list of metrics provided by this integration.
+
+### Events
+The Docker integration produces the following events:
+
+* Delete Image
+* Die
+* Error
+* Fail
+* Kill
+* Out of memory (oom)
+* Pause
+* Restart container
+* Restart Daemon
+* Update
+
+### Service Checks
+
+**docker.service_up**:
+Returns `CRITICAL` if the Agent is unable to collect the list of containers from the Docker daemon, otherwise returns `OK`.
+
+**docker.container_health**:
+This Service Check is only available for Agent v5. It returns `CRITICAL` if a container is unhealthy, `UNKNOWN` if the health is unknown, and `OK` otherwise.
+
+**docker.exit**:
+Returns `CRITICAL` if a container exited with a non-zero exit code, otherwise returns `OK`.
+
+**Note**: To use `docker.exit`, add `collect_exit_code: true` in your [Docker YAML file][20] and restart the Agent.
+
+## Troubleshooting
+Need help? Contact [Datadog support][21].
+
+## Further Reading
+### Knowledge Base
+
+* [Compose and the Datadog Agent][22]
+* [DogStatsD and Docker][23]
+
+### Datadog Blog
+
+Learn more about how to monitor Docker performance metrics with [our series of posts][24]. We detail the challenges when monitoring Docker, its key performance metrics, how to collect them, and lastly how the largest TV and radio outlet in the U.S. monitors Docker using Datadog.
+
+We've also written several other in-depth blog posts to help you get the most out of Datadog and Docker:
+
+* [How to Monitor Docker Resource Metrics][25]
+* [How to Collect Docker Metrics][26]
+* [8 Surprising Facts about Real Docker Adoption][27]
+* [Monitor Docker on AWS ECS][28]
+* [Dockerize Datadog][29]
+* [Monitor Docker with Datadog][30]
+
```

Comment:
```suggestion
* [Compose and the Datadog Agent][22]
* [DogStatsD and Docker][23]
* [The Docker Monitoring Problem][24] (series)
* [How to Monitor Docker Resource Metrics][25]
* [How to Collect Docker Metrics][26]
* [8 Surprising Facts about Real Docker Adoption][27]
* [Monitor Docker on AWS ECS][28]
* [Dockerize Datadog][29]
* [Monitor Docker with Datadog][30]

```
We typically do not give so much context around further reading. The titles should be enough.